include/net/ethernet.h

```diff
@@ -74,7 +78,16 @@ struct net_eth_addr {
 
 #define NET_ETH_MINIMAL_FRAME_SIZE	60
 #define NET_ETH_MTU			1500
-#define NET_ETH_MAX_FRAME_SIZE	(NET_ETH_MTU + sizeof(struct net_eth_hdr))
+#define _NET_ETH_MAX_FRAME_SIZE	(NET_ETH_MTU + sizeof(struct net_eth_hdr))
+/*
+ * Extend the max frame size for DSA (KSZ8794) by one byte (to 1519) to
+ * store tail tag.
+ */
+#if IS_ENABLED(CONFIG_NET_DSA) && IS_ENABLED(CONFIG_DSA_KSZ8794)
```

Comment:
ethernet.h should stay as generic as possible, remove the second IS_ENABLED()

Reply:
Ok.